floss/plugins/library_function_plugin.py

```diff
@@ -12,12 +12,13 @@ class FunctionIsLibraryPlugin(plugin_object.GeneralPlugin):
     version = 1.0
 
     def identify(self, vivisect_workspace, function_vas):
+        function_vas = function_vas - set(vivisect_workspace.getEntryPoints())
         candidate_functions = {}
         for fva in function_vas:
             fname = vivisect_workspace.getName(fva)
-            default_name = "sub_%.8x" % fva
-            if fname != default_name:
-                self.d("Identified %s at VA 0x%08X " % (fname, fva))
+            default_name = "sub_"
+            if fname is not None and not fname.startswith(default_name):
```

Comment:
lets use `default_prefix` for this variable, instead

Reply:
https://github.com/BenjaminSoelberg/flare-floss/commit/3dceb593f36123530e91007b410ca56502f18854